packages/client-core/src/user/store/UserState.ts

```diff
@@ -16,7 +16,7 @@ const state = createState({
 
 export const userReducer = (_, action: UserActionType) => {
   Promise.resolve().then(() => userReceptor(action))
-  return state.attach(Downgraded).value
+  return state
```

Comment:
Please leave this the way it was. Need to export "downgraded" values (values that aren't wrapped in hookstate proxy objects).